mail_tracking/static/src/js/failed_message.js

```diff
@@ -0,0 +1,366 @@
+/* Copyright 2019 Alexandre Díaz
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). */
+odoo.define('mail_tracking.FailedMessage', function (require) {
+    "use strict";
+
+    var ChatAction = require('mail.chat_client_action');
+    var AbstractField = require('web.AbstractField');
+    var BasicModel = require('web.BasicModel');
+    var BasicView = require('web.BasicView');
+    var Chatter = require('mail.Chatter');
+    var utils = require('mail.utils');
+    var chat_manager = require('mail.chat_manager');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+    var time = require('web.time');
+    var session = require('web.session');
+    var config = require('web.config');
+
+    var QWeb = core.qweb;
+    var _t = core._t;
+
+
+    /* DISCUSS */
+    var failed_counter = 0;
+    var is_channel_failed_outdated = false;
+    ChatAction.include({
+        init: function () {
+            this._super.apply(this, arguments);
+            // HACK: Custom event to update messsages
+            core.bus.on('force_update_message', this, function (data) {
+                is_channel_failed_outdated = true;
+                this._onMessageUpdated(data);
+                this.throttledUpdateChannels();
+            });
+        },
+        _renderSidebar: function (options) {
+            options.failed_counter = chat_manager.get_failed_counter();
+            return this._super(options);
```

Comment:
In general terms, always use this:

```suggestion
            return this._super.apply(this, arguments);
```

The way arguments are handled in ES is so error-prone that it's the only safe way to pass them to upstream method in ES5.